packages/server/modules/workspaces/graph/resolvers/workspaces.ts

```diff
@@ -784,6 +791,28 @@ export = FF_WORKSPACES_MODULE_ENABLED
               db
             })
           })({ userId: ctx.userId!, workspaceId: args.input.workspaceId })
+        },
+        requestToJoin: async (_parent, args, ctx) => {
+          const transaction = await db.transaction()
+          return await withTransaction(
```

Comment:
How's this work? Don't you need to pass in the trx obj in place of at least some of the `db` injections in some of the factories? I would think that all of the SQL queries just sidestep the trx cause they don't reference it